/**:
  ros__parameters:
    can_interfaces:
      - name: can0
        baudrate: 500000
        vesc_ids: [1, 2]   # Left side
      - name: can1
        baudrate: 500000
        vesc_ids: [3, 4]   # Right side
    publish_rate: 50.0
